lib/core/src/lib/datatable/data-column/data-column.component.stories.ts

```diff
@@ -294,114 +355,185 @@ export default {
+                [copyContent]="copyContent"
+                [format]="format"
+                [isHidden]="isHidden"
+                [class]="cssClass"
+                [sr-title]="srTitle"
+                [currencyConfig]="currencyConfig"
+                [decimalConfig]="decimalConfig"
+                [dateConfig]="dateConfig"
+                [formatTooltip]="formatTooltip">
             </data-column>
         </data-columns>
     </adf-datatable>`
 });
 
-export const textColumn = template.bind({});
+// Text Column
+export const textColumn: Story = template.bind({});
 textColumn.args = {
-    data: 'text',
+    data: 'textMap',
     key: 'id',
     type: 'text',
     title: 'Text Column'
 };
 
-export const iconColumn = template.bind({});
+// Text Column With Custom Tooltip
+export const textColumnWithCustomTooltip: Story = template.bind({});
+textColumnWithCustomTooltip.argTypes = {
+    formatTooltip: { control: { disable: false } }
+};
+textColumnWithCustomTooltip.args = {
+    data: 'textMap',
+    key: 'id',
+    type: 'text',
+    title: 'Custom Tooltip Column',
+    formatTooltip: formatCustomTooltip
+};
+
+// Icon Column
+export const iconColumn: Story = template.bind({});
 iconColumn.argTypes = {
     copyContent: { control: { disable: true } }
 };
 iconColumn.args = {
-    data: 'icon',
+    data: 'iconMap',
     key: 'icon',
     type: 'icon',
     title: 'Icon Column'
 };
 
-export const dateColumn = template.bind({});
+// Image Column
+export const imageColumn: Story = template.bind({});
+imageColumn.argTypes = {
+    copyContent: { control: { disable: true } }
+};
+imageColumn.args = {
+    data: 'imageMap',
+    key: 'image',
+    type: 'image',
+    title: 'Image Column'
+};
+
+// Date Column
+export const dateColumn: Story = template.bind({});
 dateColumn.argTypes = {
-    format: { control: { disable: false } },
-    title: { control: { disable: true } },
     copyContent: { control: { disable: true } },
-    sortable: { control: { disable: true } },
-    draggable: { control: { disable: true } },
-    isHidden: { control: { disable: true } }
+    dateConfig: { control: { disable: false } }
 };
 dateColumn.args = {
-    data: undefined,
-    format: 'medium',
-    columns() {
-        return [{ ...data.dateColumns, format: this.format }];
-    },
-    rows: data.dateRows,
-    key: 'id',
-    type: 'date'
+    data: 'dateMap',
+    key: 'createdOn',
+    type: 'date',
+    title: 'Date Column'
 };
 
-export const fileColumn = template.bind({});
-fileColumn.argTypes = {
+// Date Column Time Ago
+export const dateColumnTimeAgo: Story = template.bind({});
+dateColumnTimeAgo.argTypes = {
+    copyContent: { control: { disable: true } },
+    format: { control: { disable: false } },
+    dateConfig: { control: { disable: false } }
+};
+dateColumnTimeAgo.args = {
+    data: 'dateTimeAgoMap',
+    key: 'modifiedOn',
+    type: 'date',
+    title: 'Date Column Time Ago',
+    format: 'timeAgo'
```

Comment:
can we use dateConfig here instead of format?
`dateConfig: { format: 'timeAgo' }`

Reply:
Does the current implementation allow for that? I think we can only use 'timeAgo' with the `format` property

Reply:
Ok, I've just tried this and works the same way - updated